compiler/env/OMRCPU.hpp

```diff
@@ -107,13 +117,19 @@ class OMR_EXTENSIBLE CPU
     */
    static TR::CPU detect(OMRPortLibrary * const omrPortLib);
 
-  /** 
+   /** 
     * @brief Returns the processor type and features that will be used by portable AOT compilations
     * @param[in] omrPortLib : the port library
     * @return TR::CPU
     */
    static TR::CPU detectRelocatable(OMRPortLibrary * const omrPortLib);
 
+   /**
+    * @brief Intialize _featureMasks to the list of processor features that will be utilized by the compiler and set _isFeatureMasksEnabled to true
+    * @return void
+    */
+   static void enableFeatureMasks();
```

Comment:
I'm not quite understanding the purpose of this API. The commit does not explain it either. I think we need a better commit description.